src/components/ProfileImage.jsx

```diff
@@ -1,29 +1,105 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
-import { fetchProfile } from "../actions/profileAction";
 import PropTypes from "prop-types";
+import { NavLink } from "react-router-dom";
+
+import { fetchProfile } from "../actions/profileAction";
+
 export class ProfileImage extends Component {
+  state = {
+    username: localStorage.getItem("username"),
+    image:
+      "https://res.cloudinary.com/dksxmwjqs/image/upload/v1545036678/ux5uy7dcbqpzu6hbsjku.jpg"
```

Comment:
@johnkegz this is a good implementation because it enables users with no profile image to have a default image

Reply:
Thanks @joshNic